src/interfaces/IMetaMorpho.sol

```diff
@@ -71,6 +71,7 @@ interface IMetaMorphoBase {
 
     /// @notice Stores the total assets managed by this vault when the fee was last accrued.
     /// @dev May be a little off `totalAssets()` after each interaction, due to some roundings.
+    /// @dev Warning: Bad debt on an enabled market might prevent fee to accrue.
```

Comment:
```suggestion
    /// @dev May be off `totalAssets()` between deposits/withdrawals due to roundings or socialized bad debt.
    /// This difference will decrease the fee accrued between 2 deposits/withdrawals.
```

Reply:
It seems `lastTotalAssets` may always be at worst greater than `totalAssets()`. It should never be smaller